addons/web/static/src/views/debug_items.js

```diff
@@ -394,6 +394,7 @@ export function manageAttachments({ component, env }) {
                 context: {
                     default_res_model: component.props.resModel,
                     default_res_id: resId,
+                    skip_res_field_check: true,
```

Comment:
I had the idea to add `["id", ">=", 0]` in the domain, it would have made the same thing as what you proposed. I discussed it with AWT this morning, and he preferred to add a context. I personally don't know which option is the best.

Reply:
Both solutions are look good to me (I was usually doing `('id', '>', 0)`).
It's a similar case to the `active_test` context key and `('active', 'in', [True, False])`. That's why both approaches are fine for me.

@ryv-odoo A preference?

Reply:
I rather prefer the context key because when/if we will remove this override, it will be easier to track the hack usage.